can/interfaces/__init__.py

```diff
@@ -10,25 +10,29 @@
 
 # interface_name => (module, classname)
 BACKENDS = {
-    'kvaser':           ('can.interfaces.kvaser',           'KvaserBus'),
-    'socketcan':        ('can.interfaces.socketcan',        'SocketcanBus'),
-    'serial':           ('can.interfaces.serial.serial_can','SerialBus'),
-    'pcan':             ('can.interfaces.pcan',             'PcanBus'),
-    'usb2can':          ('can.interfaces.usb2can',          'Usb2canBus'),
-    'ixxat':            ('can.interfaces.ixxat',            'IXXATBus'),
-    'nican':            ('can.interfaces.nican',            'NicanBus'),
-    'iscan':            ('can.interfaces.iscan',            'IscanBus'),
-    'virtual':          ('can.interfaces.virtual',          'VirtualBus'),
-    'neovi':            ('can.interfaces.ics_neovi',        'NeoViBus'),
-    'vector':           ('can.interfaces.vector',           'VectorBus'),
-    'slcan':            ('can.interfaces.slcan',            'slcanBus'),
-    'canalystii':       ('can.interfaces.canalystii',       'CANalystIIBus'),
-    'systec':           ('can.interfaces.systec',           'UcanBus')
+    "kvaser": ("can.interfaces.kvaser", "KvaserBus"),
```

Comment:
Do we really want this new formatting here? Or is there no option to exclude a block?

Reply:
black provides a way to exclude sections from formatting, but IMO that kind of goes against the point of having a formatter.

If someone feels strongly about keeping it untouched by the formatter, I can add

```python
# fmt: off

# ...

# fmt: on
```